.github/workflows/check-extension-case.yml

```diff
@@ -0,0 +1,34 @@
+name: CheckExtensionCase
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+          
+      - id: file_changes
+        uses: trilom/file-changes-action@v1.2.3
+
+      - name: test
+        run: |
+          FILES=${{steps.file_changes.outputs.files_added}}
+
+          if [ ${#FILES} -ge 3 ];
+          then
+              IFS=", " read -a myarray <<< $FILES
+
+              for filename in "${myarray[@]}" 
+              do
+                  pureFile=$(echo $filename | sed 's/"//g' | sed 's/]//g' | grep -o '...$')
+                  if [ $pureFile == "GIF" ] || [ $pureFile == "SVG" ];
```

Comment:
use regex to check for things like .giF or .Gif as well.

Reply:
Done - The whole extension has to be lowercase